front-end/src/components/analysis-page/IntentLister.jsx

```diff
@@ -118,10 +129,10 @@ class IntentLister extends Component {
     render() {
         const MAX_CURRENT_INTENTS = 3;
         let currentIntents = [];
-        let curr = this.state.index;
+        let curr = this.props.initialIndex;
```

Comment:
```suggestion
        let curr = this.props.index;
```

We should change the name of this prop to "index" instead of initialIndex, since with the new changes, "initial" is no longer an accurate description of what the variable represents.

Reply:
It was meant as the index we were starting at as we loaded onto analysis page. I agree with William on the renaming :+1: now that we are in redux

Reply:
^^^ yes actually I named it that a couple weeks ago LOL. i don't really remember why i did that, i think it was to make testing easier in some way.